syntax = "proto2";

package tbclient;

import "tbclient/ActPost.proto";
import "tbclient/AddPostList.proto";
import "tbclient/Agree.proto";
import "tbclient/DealInfo.proto";
import "tbclient/Lbs.proto";
import "tbclient/OriginThreadInfo.proto";
import "tbclient/PbContent.proto";
import "tbclient/PbPostZan.proto";
import "tbclient/PbPresent.proto";
import "tbclient/SignatureData.proto";
import "tbclient/SimpleForum.proto";
import "tbclient/SkinInfo.proto";
import "tbclient/SubPost.proto";
import "tbclient/TPointPost.proto";
import "tbclient/TailInfo.proto";
import "tbclient/TogetherHi.proto";
import "tbclient/User.proto";
import "tbclient/VideoInfo.proto";
import "tbclient/Zan.proto";

message Post {
  optional ActPost act_post = 27;
  optional AddPostList add_post_list = 16;
  optional uint32 add_post_number = 20;
  optional Agree agree = 37;
  repeated string arr_video = 6;
  optional User author = 23;
  optional int64 author_id = 19;
  optional string bimg_url = 17;
  repeated PbContent content = 5;
  repeated TailInfo ext_tails = 32;
  optional uint32 floor = 3;
  optional SimpleForum from_forum = 38;
  optional TogetherHi high_together = 33;
  optional uint64 id = 1;
  optional int32 img_num_abtest = 41;
  optional string ios_bimg_format = 18;
  optional uint32 is_bub = 11;
  optional int32 is_fold = 43;
  optional int32 is_hot_post = 31;
  optional uint32 is_ntitle = 10;
  optional int32 is_post_visible = 39;
  optional uint32 is_voice = 9;
  optional uint32 is_vote = 8;
  optional Lbs lbs_info = 7;
  optional string lego_card = 36;
  optional int32 need_log = 40;
  optional OriginThreadInfo origin_thread_info = 42;
  optional DealInfo pb_deal_info = 35;
  optional PbPostZan post_zan = 30;
  optional PbPresent present = 28;
  optional SignatureData signature = 21;
  optional SkinInfo skin_info = 34;
  optional int32 storecount = 25;
  optional SubPost sub_post_list = 15;
  optional uint32 sub_post_number = 13;
  optional TailInfo tail_info = 22;
  optional uint32 time = 4;
  optional string time_ex = 14;
  optional string title = 2;
  optional TPointPost tpoint_post = 26;
  optional VideoInfo video_info = 29;
  optional string vote_crypt = 12;
  optional Zan zan = 24;
}
